S2VX.Game/Editor/S2VXEditor.cs

```diff
@@ -185,6 +185,9 @@ public class S2VXEditor : CompositeDrawable {
                     break;
                 case Key.Right:
                     NotesTimeline.SnapToTick(false);
+                    break;
+                case Key.Delete:
+                    ToolState.OnToolKeyDown(e);

```

Comment:
As discussed, this should be moved to the top of the handler so that ToolState can handle the key and return back whether a key was processed.